### DISCUSSION TITLE
Comptes Client

### DISCUSSION THREAD

**Discussion by AGareil** [bgo-bioimagerie/platformmanager#595] [Ideas]
Bonjour,

Dans la nouvelle version de PFM du 28 février (2.4) la présentation des comptes clients a été modifié.

Donc maintenant c'est chouette, on peut voir les personnes associées à un compte client et les rajouter si on ne les trouve pas.
Mais, pour associer une personne à un compte client on a un clic en plus à faire.
Avant, on entrait une personne, on allait dans les utilisateurs puis on cliquait sur "Compte client" et on était directement sur la page pour leur associer un compte.
Maintenant on doit aller sur "Accès" puis l'onglet "Compte" et là on peut rentrer le compte.
Ca devrait être de plus en plus calme sur ECOBIO, mais aux périodes d'arrivée de stagiaire le clic en plus se sent.
Alors pour ça je n'ai pas d'idée c'est juste un ressenti.

Par contre, la nouvelle moulure marche bien quand on a un compte et que l'on veut savoir qui est dessus ou pas en ayant un nom à vérifier. A moins que je sois passé à côté, on n'a pas la possibilité d'un coup d'oeil de voir qui a ou pas son nom associé 
Est ce qu'il serait envisageable de mettre  dans le tableau des utilisateurs (où l'on a accès, impersonate, nom, prénom, mail...) une colonne "Compte client" avec un oui quand il en ont un associé et non quand ce n'est pas le cas? Ou une croix? Ou le compte client associé, mais ça fera un peu trop chargé peut être?

Personnelement, c'est ce qui m'est le plus remonté comme "bug" les gestionnaires des calendriers sont perdu avec juste cet élément et pourtant pour les statistiques je suis sûre qu'ils vont beaucoup compter dessus.

Comme ce n'est pas un bugg, je mets ça dans idées.

Merci,

Agnès

**Comment by osallou** [comment]
afficher le compte client surchargerait l'interface, un utilisateur peut avoir pls comptes clients.

mais il serait envisageable d'afficher si oui ou non il est associé à au moins un compte.
